Description: Disable ieee_arithmetic.F95 as not compiling on gfortran at the moment
Author: Alastair McKinstry <mckinstry@debian.org>
Forwarded: no
Last-Updated: 2018-11-29

Index: flang-20180921/runtime/flang/CMakeLists.txt
===================================================================
--- flang-20180921.orig/runtime/flang/CMakeLists.txt
+++ flang-20180921/runtime/flang/CMakeLists.txt
@@ -455,9 +455,9 @@ FOREACH(file ${FTN_SUPPORT_DESC_DEP})
     LIST(APPEND FTN_SUPPORT_I8 ${I8_FILE})
 ENDFOREACH(file)
 
-LIST(APPEND FTN_SUPPORT_DESC_DEP ieee_arithmetic.F95)
+#LIST(APPEND FTN_SUPPORT_DESC_DEP ieee_arithmetic.F95)
 #LIST(APPEND FTN_SUPPORT_DESC_DEP ieee_exceptions.F95)
-LIST(APPEND FTN_SUPPORT_I8 "${I8_FILES_DIR}/ieee_arithmetic.F95")
+#LIST(APPEND FTN_SUPPORT_I8 "${I8_FILES_DIR}/ieee_arithmetic.F95")
 #LIST(APPEND FTN_SUPPORT_I8 "${I8_FILES_DIR}/ieee_exceptions.F95")
 
 add_flang_library(flang_static
@@ -544,7 +544,7 @@ set_source_files_properties(
 
 # State a dependency on the module
 set_source_files_properties(
-  ieee_arithmetic.F95
+#  ieee_arithmetic.F95
 #  ieee_exceptions.F95
   PROPERTIES
   OBJECT_DEPENDS ${CMAKE_Fortran_MODULE_DIRECTORY}/iso_c_binding.mod
@@ -552,7 +552,7 @@ set_source_files_properties(
 
 # State a dependency on the module
 set_source_files_properties(
-  ${I8_FILES_DIR}/ieee_arithmetic.F95
+#  ${I8_FILES_DIR}/ieee_arithmetic.F95
 #  ${I8_FILES_DIR}/ieee_exceptions.F95
   PROPERTIES 
   COMPILE_DEFINITIONS DESC_I8
